tools/linter/linter.py

```diff
@@ -122,10 +162,16 @@ def _lint_builtin_functions(n):
     if isinstance(n, ast.Call) and isinstance(n.func, ast.Name) and n.func.id in BUILTIN_FUNCTIONS:
         if n.args or n.starargs:
             yield n.lineno, NON_KEYWORD_CALL
-        args = {arg['name'] for arg in BUILTIN_FUNCTIONS[n.func.id]['args']}
+        args = BUILTIN_FUNCTIONS[n.func.id]['args']
         for kwd in n.keywords or []:
             if kwd.arg not in args and not kwd.arg.startswith('_'):
                 yield kwd.value.lineno, INCORRECT_ARGUMENT
+        # Don't check kwargs if the caller is doing a **kwargs into it, assume that'll take care of it.
+        if not n.kwargs:
+            kwds = {kwd.arg for kwd in n.keywords or []}
```

Comment:
if `n.keywords` is `None` then it's not iterable and we'd get an exception.

(Note that the if statement above is `kwargs` not `keywords`...)